static/app/components/organizations/pageFilterBar.tsx

```diff
@@ -110,7 +111,19 @@ const pageFilterBarStyles = (p: {theme: Theme; condensed?: boolean}) => css`
   }
 `;
 
+const getChildTransforms = (count: number) => {
+  return Array.from(
+    {length: count},
+    (_, index) => css`
+      div:nth-of-type(${index + 1}) > button {
```

Comment:
the `div` here feels a bit brittle. should we just use `:nth-of-type(${index + 1}) > button`?

Reply:
Eh, I think it's fine. We can revisit.